kotlin/build.gradle.kts

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation("com.aventrix.jnanoid:jnanoid:2.0.0")
     implementation("com.squareup.okhttp3:okhttp")
     implementation("com.google.code.gson:gson:2.8.9")
+    implementation("ch.qos.logback:logback-classic:1.2.10")
```

Comment:
what's this package used for?

Reply:
The `jose` relies on this package to do some logging.
If we miss this package, we will get an error when using jose.